soc/nxp/layerscape/Kconfig.soc

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright 2024 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SOC_FAMILY_LAYERSCAPE
+	bool
+
+config SOC_FAMILY
+	default "nxp_layerscape" if SOC_FAMILY_LAYERSCAPE
```

Comment:
either:
```suggestion
config SOC_FAMILY_NXP_LAYERSCAPE
	bool

config SOC_FAMILY
	default "nxp_layerscape" if SOC_FAMILY_NXP_LAYERSCAPE
```
or
```suggestion
config SOC_FAMILY_LAYERSCAPE
	bool

config SOC_FAMILY
	default "layerscape" if SOC_FAMILY_LAYERSCAPE
```

Reply:
Changed to use SOC_FAMILY_NXP_LAYERSCAPE, thanks.